src/devtools/client/debugger/src/actions/sources/symbols.ts

```diff
@@ -24,14 +24,14 @@ async function doSetSymbols(source: Source, thunkArgs: { dispatch: AppDispatch }
 }
 
 export const setSymbols = memoizeableAction("setSymbols", {
-  getValue: ({ source }: { source: Source }, thunkArgs) => {
+  getValue: ({ source }, thunkArgs) => {
     const symbols = selectors.getSymbols(thunkArgs.getState(), source);
     if (!symbols || symbols.loading) {
       return null;
     }
 
     return fulfilled(symbols);
   },
-  createKey: ({ source }) => source.id,
+  createKey: ({ source }: { source: SourceDetails }) => source.id,
   action: ({ source }, thunkArgs) => doSetSymbols(source, thunkArgs),
 });
```

Comment:
cant wait to kill symbols
https://linear.app/replay/issue/FE-446/start-using-symbols-from-the-server

Reply:
Yeah, this may be the last remaining use of `memoizableAction` - I'd have to see what else is left.